apps/beeai-ui/src/modules/agents/components/AgentTags.tsx

```diff
@@ -46,7 +46,7 @@ export function AgentTags({ agent, className }: Props) {
     .filter(isNotNull)
     .map((value) => <AgentTag key={value} name={value} />);
 
-  return <TagsList tags={tags} className={className} />;
+  return tags.length > 0 ? <TagsList tags={tags} className={className} /> : null;
```

Comment:
How about `<TagsList>` decides what to render based on the _length_ of the **_tags_**?